### PR TITLE
Remove unused assets | Replace incorrect image formats | Enable acces…

### PR DESCRIPTION
**Pull request by KevJ2021**
…sibility

### CODE REVIEW

File: _includes/2up.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h2 class="text-colour-{{include.textColor}}"><span class="dot"></span>{{include
   </div>
   <div class="t-half image">
     {% if include.image %}
-      <img src="{{include.image}}" alt="young woman in red sari standing">
+      <img src="{{include.image}}" alt="young woman in red sari standing" title="young woman in red sari standing">
```
Comment (a-bresler) [inline _includes/2up.html:30]: Are these titles visible to the user? Wondering if it is worth capitalising the start of the sentence or using some labelling syntax (capitalising each word?)

**Review by a-bresler — APPROVED** [review]
LGTM! Just a question on the titles 🙂 

giphy (1)

File: _includes/2up.html (hunk 1 above)
Comment (KevJ2021) [reply _includes/2up.html:30]: Good catch! Yes, they are visible to the user depending on which browser they use. Agreed, we should likely use Title case for both the Alt attribute and Title case, I kept them the same for now as I'm unsure if there was a reason for the original Alt tag being left in lower case.

I think we should create another task, and take a deeper dive into accessibility on the website, as I've noticed a few places lacking in accessibility optimization.

More of a future reference on best practices: https://www.deque.com/blog/great-alt-text-introduction/ | https://www.w3.org/WAI/fundamentals/accessibility-intro/ 

Thanks @a-bresler!

Comment (a-bresler) [reply _includes/2up.html:30]: Happy 👌  Thanks for the extra info as well! 😁

**Pull request merged**